config/crds/tenancy.kcp.dev_clusterworkspaces.yaml

```diff
@@ -68,17 +68,26 @@ spec:
               readOnly:
                 type: boolean
               type:
-                default: Universal
                 description: "type defines properties of the workspace both on creation
                   (e.g. initial resources and initially installed APIs) and during
-                  runtime (e.g. permissions). \n The type is a reference to a ClusterWorkspaceType
-                  in the same workspace with the same name, but lower-cased. The ClusterWorkspaceType
-                  existence is validated at admission during creation, with the exception
-                  of the \"Universal\" type whose existence is not required but respected
-                  if it exists. The type is immutable after creation. The use of a
-                  type is gated via the RBAC clusterworkspacetypes/use resource permission."
-                pattern: ^[A-Z][a-zA-Z0-9]+$
-                type: string
+                  runtime (e.g. permissions). If no type is provided, the default
+                  type for the workspace in which this workspace is nesting will be
+                  used. \n The type is a reference to a ClusterWorkspaceType in the
+                  listed workspace, but lower-cased. The ClusterWorkspaceType existence
```

Comment:
this reads a little strange. The type is a reference to a (lower-cased) ClusterWorkspaceType.

Reply:
This predates me

Reply:
indeed. My former me probably wrote that.